ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/macro/MacroRegistryImpl.java

```diff
@@ -39,9 +38,7 @@ public MacroRegistryImpl() {
     public void register(Set<Macro> macros) {
         for (Macro macro : macros) {
             final String name = macro.getName();
-            if (this.macros.containsKey(name)) {
-                Log.warn(MacroRegistryImpl.class, "Command macro '" + name + "' is already registered.");
```

Comment:
There are huge amount of messages in the console when restarting the workspace.
I had full console of the following
  Command macro ... is already registered.
  Command macro ... is already registered.
  Command macro ... is already registered.
  Command macro ... is already registered.